src/components/taskDetails/DetailsColor.tsx

```diff
@@ -0,0 +1,7 @@
+const setColor: Record<string, string> = {
```

Comment:
Change the type to `ts` and name to something else like `taskPriorityColors`

Reply:
Done 👍🏻